src/SpectatorView.Unity/Assets/SpectatorView/Scripts/ScreenRecording/Plugins/Android/AndroidManifestTemplate.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="UNITY WILL REPLACE" xmlns:tools="http://schemas.android.com/tools" android:installLocation="preferExternal">
+  <supports-screens android:smallScreens="true" android:normalScreens="true" android:largeScreens="true" android:xlargeScreens="true" android:anyDensity="true" />
+  <application android:theme="@style/UnityThemeSelector" android:icon="@mipmap/app_icon" android:label="@string/app_name" android:isGame="true" android:banner="@drawable/app_banner">
+    <activity android:label="@string/app_name" android:screenOrientation="portrait" android:launchMode="singleTask" android:configChanges="mcc|mnc|locale|touchscreen|keyboard|keyboardHidden|navigation|orientation|screenLayout|uiMode|screenSize|smallestScreenSize|fontScale|layoutDirection|density" android:hardwareAccelerated="false" android:name="Microsoft.MixedReality.SpectatorView.Unity.ScreenRecorderActivity">
```

Comment:
Can user permissions be declared in this template? We were declaring WRITE_EXTERNAL_STORAGE and RECORD_AUDIO in our previously existing manifest.

Reply:
Both of those will be automatically added by Unity to the final manifest.  I debated putting comments in this template to explain, but then those comments end up in the final manifest, too, so I opted to leave them out.  I'm happy to put comments back into the manifest though if you'd prefer.